android/app/src/main/kotlin/net/defined/mobile_nebula/MainActivity.kt

```diff
@@ -75,6 +80,9 @@ class MainActivity: FlutterActivity() {
                 "nebula.verifyCertAndKey" -> nebulaVerifyCertAndKey(call, result)
 
                 "dn.enroll" -> dnEnroll(call, result)
+                "dn.getPollToken" -> dnGetPollToken(call, result)
```

Comment:
if you want to play with this on iOS, these are the bindings you need to implement.

Below, you can see I also had to make a version of the binding channel that runs on the background executor, to avoid blocking the UI thread. You probably need both.